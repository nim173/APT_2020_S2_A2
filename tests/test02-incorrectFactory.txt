BURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLY
A
B
TURN 2 B 1
TURN 7 B 1