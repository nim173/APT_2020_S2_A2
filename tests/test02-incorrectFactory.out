
Invalid turn with respect to game context at: 
2: TURN 7 B 1

Factories:
0: F Y U R 
1: B U R L 
2: 
3: L Y B U 
4: R L Y B 
5: U R L Y 


Mosaic for A:
1:         B || . . . . .
2:       . . || . . . . .
3:     . . . || . . . . .
4:   . . . . || . . . . .
5: . . . . . || . . . . .
broken: 


Mosaic for B:
1:         . || . . . . .
2:       . . || . . . . .
3:     . . . || . . . . .
4:   . . . . || . . . . .
5: . . . . . || . . . . .
broken: 

